app/mailers/invoice_error_mailer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+class InvoiceErrorMailer < ApplicationMailer
+  def report(duplicate_ids:)
+    @duplicate_ids = duplicate_ids
+    mail(to: "cac9@princeton.edu, mzelesky@princeton.edu, kr2@princeton.edu", subject: 'PeopleSoft Duplicate Invoices')
```

Comment:
The invoices are generated by Alma and sent to PeopleSoft; they are not sent from PeopleSoft. So this should have a subject of 'Alma Duplicate Invoices.'

Reply:
@mzelesky maybe I have this incorrect, but the data is coming from PeopleSoft that we are checking (Correct?). I am looking at Duplicate data from the PeopleSof Report.  The duplicate data is not from Alma, or is Alma creating a Duplicate?
Should this really say `PeopleSoft Duplicate Transaction`?

Reply:
Yes. `PeopleSoft Duplicate Fund Transactions`.